core/whistle-1.0.0/src/wh_cache.erl

```diff
@@ -143,6 +145,11 @@ start_link(Name, ExpirePeriod, Props) ->
                                    )
     end.
 
+-spec maybe_add_db_binding(wh_proplists()) -> wh_proplists().
+maybe_add_db_binding([[]]) -> [[]];
```

Comment:
@Dinkor this should be wh_proplist() and function clause should match [] and not [[]]. we are matching empty BindingProps , not BindingProps with one element that is an empty list.